vipps-recurring-api.md

```diff
@@ -635,28 +650,38 @@ a description with follow format `{agreement.ProductName} - {charge.description}
 ### Charge times
 
 Charge _attempts_ are primarily made two times during the day: 07:00 and 15:00 UTC.
+Vipps may change this without notice.
 The processing of charges typically takes around one hour, however this varies and we do not guarantee any time.
 This is the same both for our production and test environment.
 Subsequent attempts are made according to the `retryDays` specified.
 
-**Note:** Payments _might_ get processed any time during the day (07:00 UTC - 23:59 UTC) due to special circumstances requiring it.
+**Note:** Payments _might_ get processed any time during the day
+(07:00 UTC - 23:59 UTC) due to special circumstances requiring it.
 
 **Note:** Since payments _can_ be processed any time (07:00UTC - 23:59 UTC) it is advisable to fetch the charge at/after 00:00 UTC the day after the last retry day to be sure you get the last status.
 
-### Charge retries
+When a charge has reached its `due` date, the status of the charge will be
+`PROCESSING` until the charge is successful, for as long as the merchant has
+specified with `retryDays`.
```

Comment:
We are not 100% sure we want to do this. Merchants are currently used to charges staying as DUE for the retry-period, so it might not make sense to start setting them as PROCESSING

Reply:
See the long(!) discussions on Slack.